cmd/bnscli/cmd_gov.go

```diff
@@ -485,6 +485,7 @@ Creates a new version for an existing election rule. The new version is used for
 		durationFl    = fl.Int("voting-period", 0, "Duration in seconds how long the voting period will take place")
 		numeratorFl   = fl.Int("threshold-numerator", 0, "The top number of the fraction.")
 		denominatorFl = fl.Uint("threshold-denominator", 0, "The bottom number of the fraction")
+		quorumFl      = flFraction(fl, "quorum", "0/1", "New quorum fraction in format <numerator>/<denominator>. Zero quorum deletes the value.")
```

Comment:
I am skeptical of the default.
If I don't provide a value, it should be a no-op, not changing the quroum